src/options.h

```diff
@@ -2272,6 +2272,11 @@ nl_fdef_brace;
 extern Option<iarf_e>
 nl_fdef_brace_cond;
 
+// Add or remove newline between last constructor initializer list argument
+// and '{'. Overrides nl_fdef_brace and nl_fdev_brace_cond.
```

Comment:
Nitpicky:

```suggestion
// Add or remove newline between last constructor initializer list argument and
// '{'. Overrides nl_fdef_brace and nl_fdev_brace_cond.
```

...but I am *far* less averse to breaking early if you think it reads better as-is versus going over length. Otherwise this is pretty good documentation (which is not always the case); thanks!